app/uk/gov/hmrc/helptosavefrontend/views/register/not_eligible.scala.html

```diff
@@ -26,44 +26,66 @@
 @uk.gov.hmrc.helptosavefrontend.views.html.main_template(
     title = Messages("hts.core.not-eligible.page-title"),
     bodyClasses = None) {
+
   <h1 class="heading-large" id="hts-not-eligible-title">@Messages("hts.core.not-eligible.h1")</h1>
 
-  @if(ineligibilityReason === EntitledToWTCNoTCAndInsufficientUC || ineligibilityReason === NotEntitledToWTCAndUCInsufficient){
-    <p>@Messages("hts.core.not-eligible.uc-insufficient-income.p1")</p>
 
-    <p>@Messages("hts.core.not-eligible.uc-insufficient-income.p-2-1")
+  @if(ineligibilityReason === NotEntitledToWTCAndUCInsufficient) {
```

Comment:
Just to double check, out of all thes ineligibility reasons:
```scala
  case object EntitledToWTCNoTCAndNoUC extends IneligibilityReason
  case object EntitledToWTCNoTCAndInsufficientUC extends IneligibilityReason
  case object NotEntitledToWTCAndNoUC extends IneligibilityReason
  case object NotEntitledToWTCAndUCInsufficient extends IneligibilityReason
  case object NotEntitledToWTCAndUnknownUC extends IneligibilityReason
 case object InvalidNino extends IneligibilityReason
```
 do we have the right if statements?